src/components/EnsAddress.tsx

```diff
@@ -13,27 +13,33 @@ interface EnsAddressProps {
 function EnsAddressSuspended({
   address,
   truncate,
-}: EnsAddressProps & { truncate?: boolean }) {
+  truncateSize,
```

Comment:
If you're using `truncateSize`, this means that you want to truncate, so you don't need two props, I guess

Reply:
Agreed, otherwise looks good

Reply:
This is not true, `truncate` is needed for `ensName` and only when the screen size is large :)